extensions/api/api-core/build.gradle.kts

```diff
@@ -28,12 +28,12 @@ dependencies {
     implementation(project(":common:util"))
     implementation("jakarta.ws.rs:jakarta.ws.rs-api:${rsApi}")
     implementation("jakarta.validation:jakarta.validation-api:${jakartaValidationApi}")
+    implementation("org.glassfish.jersey.ext:jersey-bean-validation:${jerseyVersion}") //for validation
```

Comment:
This could be removed from all the extension where it's defined as `testRuntimeOnly`